session-01/validators.js

```diff
@@ -17,7 +25,20 @@ function validUsername(username) {
   - password must contain at least 1 letter, 1 number, and 1 special character
 */
 function validPassword(password) {
-  return;
+  let SpecialChars = /[!@#$%^&*.?]/; 
+  let UpperLetter = /[a-z]/;
+  let LowerLetter = /[A-Z]/;
+  let Num = /[0-9]/;
+  if(password.length>9&&password.length<65){
+    if(password.search(SpecialChars)!=-1){
+      if(password.search(UpperLetter)!=-1 || password.search(LowerLetter)!=-1){
+        if(password.search(Num)!=-1){
+          return true;
+        }
+      }
+    }
+  }
+  return false;
```

Comment:
This code is correct, but for clarity in more complex code, consider using if-statement guard clauses like in this example: https://github.com/CUNYTechPrep/js-summer-prep-2024/blob/main/session-01/validators.solution.js#L10-L16